config/config.go

```diff
@@ -58,6 +58,10 @@ type Logging struct {
 type Verifier struct {
 	// did to be used by the verifier
 	Did string `mapstructure:"did"`
+	// Identification to be used for the verifier
+	ClientIdentification ClientIdentification `mapstructure:"clientIdentification"`
+	// supported request modes - currently 'urlEncoded', 'byValue' an 'byReference' are available. In case of byValue, the keyPath has to be set.
```

Comment:
```suggestion
	// supported request modes - currently 'urlEncoded', 'byValue' and 'byReference' are available. In case of byValue, the keyPath has to be set.
```